packages/protocol/contracts/governance/Validators.sol

```diff
@@ -402,6 +402,8 @@ contract Validators is
     require(uptime <= FixidityLib.fixed1().unwrap(), "Uptime cannot be larger than one");
     uint256 numerator;
     uint256 denominator;
+    uptime = uptime + 6250000000000000000000 > FixidityLib.fixed1().unwrap() ? FixidityLib.fixed1().unwrap() : uptime + 6250000000000000000000;
+    uint256 updated = uptime + grace_period.unwrap();
```

Comment:
@viktorbunin 
It's failing because this `grace_period` is not defined
I think that this was the first approach of the implementation and it's a line that probably should be removed, right?

Reply:
Yes, removed, thank you!